templates/plugin/virt.conf.erb

```diff
@@ -19,6 +19,11 @@
 <% if @ignore_selected != nil -%>
   IgnoreSelected <%= @ignore_selected %>
 <% end -%>
+<%- if scope.lookupvar('collectd::collectd_version_real') and (scope.function_versioncmp([scope.lookupvar('collectd::collectd_version_real'), '5.5']) >= 0) -%>
```

Comment:
i can't tell why yet, but i don't like this line one bit

Reply:
That's fair :) I only added it because I was trying to get the tests to make sense. The `PluginInstanceFormat` was only added as of 5.5.0 fully, and the template would include it when generating the resulting configuration, which was causing the tests to fail.

I haven't used puppet in several years, so I'm honestly not sure the appropriate approach here.

Reply:
Any suggestions on what I should do here if anything?

Reply:
no
not before we convert this epp

Reply:
yes it's ugly, but erb is pretty limited :sadface: